talpid-core/Cargo.toml

```diff
@@ -59,14 +59,14 @@ byteorder = "1"
 nftnl = { version = "0.6", features = ["nftnl-1-1-0"] }
 mnl = { version = "0.2.0", features = ["mnl-1-0-4"] }
 which = { version = "4.0", default-features = false }
-tun = "0.5"
+tun = { git = "https://github.com/Ch00k/rust-tun.git", branch = "bump-ioctl" }
```

Comment:
We want to avoid git dependencies as far as we can. Can you try getting this merged upstream and published?

Reply:
Sorry. I see now that you linked to your PR. Yes. Let's hope @meh can merge and publish that :3

Reply:
This has been published :tada:

Reply:
Fixed